modelscan/cli.py

```diff
@@ -115,6 +115,7 @@ def scan(
         with open(settings_file_path, encoding="utf-8") as sf:
             settings = parse(sf.read()).unwrap()
             click.echo(f"Detected settings file. Using {settings_file_path}. \n")
+            sf.close()
```

Comment:
The `close` should be redundant as the context manager should close it.

Reply:
Thank you Faisal, I have updated the PR, removed the redundant close & also ran make format to ensure formatting is corrected.